app/controllers/api/v1/statuses_controller.rb

```diff
@@ -4,7 +4,7 @@ class Api::V1::StatusesController < ApiController
   before_action -> { doorkeeper_authorize! :read }, except: [:create, :destroy, :reblog, :unreblog, :favourite, :unfavourite]
   before_action -> { doorkeeper_authorize! :write }, only:  [:create, :destroy, :reblog, :unreblog, :favourite, :unfavourite]
   before_action :require_user!, except: [:show, :context, :card, :reblogged_by, :favourited_by]
-  before_action :set_status, only:      [:show, :context, :card, :reblogged_by, :favourited_by]
+  before_action :set_status, only:      [:show, :update, :context, :card, :reblogged_by, :favourited_by]
```

Comment:
Why is there suddenly a `StatusController#update` method? I don't remember that being in there (and I am against status editing after-the-fact)